src/Utilities/Compiler/WellKnownTypes.cs

```diff
@@ -9,6 +9,7 @@ internal static class WellKnownTypes
     {
         public const string SystemSecurityCryptographyCipherMode = "System.Security.Cryptography.CipherMode";
         public const string SystemNetSecurityRemoteCertificateValidationCallback = "System.Net.Security.RemoteCertificateValidationCallback";
+        public const string SystemAppContext = "System.AppContext";
```

Comment:
Would it be possible to avoid adding constant strings to this type? @dotpaul and I worked on PRs in features/dataflow to move constant string names into a separate type.

Reply:
See https://github.com/dotnet/roslyn-analyzers/pull/2127 and https://github.com/dotnet/roslyn-analyzers/pull/2137

Reply:
I was thinking about doing the same in master, but haven't gotten to it yet.